assistant/src/test/java/com/ibm/watson/assistant/v2/AssistantServiceTest.java

```diff
@@ -58,9 +58,12 @@ public void setUp() throws Exception {
     super.setUp();
     String apiKey = getProperty("assistant.apikey");
     assistantId = getProperty("assistant.assistant_id");
+    // String iam_url = getProperty("assistant.auth_url");
```

Comment:
No commented code. NEVER

Reply:
fixed in [0c988ba64400180d37c77d34269cabb20d50e53a](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/0c988ba64400180d37c77d34269cabb20d50e53a)